syntax = "proto2";

package fueling.common.record.kinglong.proto.cybertron;

message Rule {
    optional string channel_name = 1;
    optional int32 hz = 2;
    optional double duration = 3;
    repeated string depends = 4;
    optional int32 ignore = 5;
    optional int32 only_once = 6;
};

message RuleDesc {
    repeated Rule rules = 1;
};
